fastscanner/services/scanners/lib/parabolic.py

```diff
@@ -75,7 +77,7 @@ async def scan(
             freq,
             [atr_iday, cum_low, cum_volume],
         )
-        df = df.loc[df.index.time <= self._end_time]  # type: ignore
+        df = df.loc[(df.index.time >= self._start_time) & (df.index.time <= self._end_time)]  # type: ignore
```

Comment:
As specified in the ticket, we need to filter by start_time before adding the cumulative indicators.
In this case you need to:
1. calculate only with the atr_iday
2. filter by start_time, end_time
4. check if the df is empty
3. extend the df with cum_low and cum_volume.

This came from a client's requirement to ignore premarket data for parabolic indicators

Reply:
You will need to fix this for both parabolic indicators